java-source/build.gradle

```diff
@@ -32,15 +32,17 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.11'
```

Comment:
Corda has a transitive dependency on JUnit 4.12 via `junit-quickcheck-generators`. Upgrade to JUnit 4.12 here too, to prevent conflict?